.github/workflows/pr.yml

```diff
@@ -0,0 +1,52 @@
+name: Generate JSON and Save as Artifacts
+
+on:
+  pull_request:
+     types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    name: generate-json-and-save-as-artifacts
+    runs-on: windows-latest
```

Comment:
Does this need to be on windows? It seems like it should be able to run on `ubuntu-latest`. Windows runners use 2x the CI minutes of Linux runners.